### ISSUE TITLE
!todo chequy of fess has bad number of rows

### ISSUE THREAD

**Issue by karlwilcox** [drawshield/Drawshield-Code#212]
!todo chequy of fess has bad number of rows
https://drawshield.net/cache/932bf3e3e62ea6b5e6888436f89caac7.png

**Comment by ericscheid** [comment]
per https://drawshield.net/reference/parker/c/chequy.html,
> a fesse or other ordinary when blazoned chequy must contain three rows of squares, for if there be but one, the ordinary will be compony, and if but two, counter-compony

`argent, a fess chequy gules and sable`
![image](https://user-images.githubusercontent.com/2605467/126802978-3c6ca2a4-78a2-4d23-b0f5-8c7d7dd159f1.png)

**Comment by karlwilcox** [comment]
Yes, good point. At the moment chequy is just the chequy treatment used as a mask (you can demonstrate with _argent 3 bars chequy sable and gules_ - you can see the underlying grid pattern)

This needs special handling so is new code rather than a small fix.
